src/docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -83,10 +83,22 @@ For some metric alerts, you can set the event type that you want to be alerted a
 
 ### Tags & Properties
 
-Add filters in the provided input to narrow down what you'll be alerted about, such as URL tags or other event properties. Available properties depend on your alert's event type:
+Add filters in the provided input to narrow down what you'll be alerted about, such as transaction tags or other event properties. Available properties depend on your alert's event type:
 
 - for **error** events, all error properties are available. See [Searchable Properties](/product/sentry-basics/search/searchable-properties/#event-properties) for a full list
```

Comment:
```suggestion
- For **error** events, all error properties are available. See [Searchable Properties](/product/sentry-basics/search/searchable-properties/events/#searchable-properties) for a full list.
```

Need to update the link since searchable properties was broken down into 4 child pages. Not sure if there's a better link for error event properties?